design/ocwg-design-decisions.md

```diff
@@ -180,10 +131,9 @@ minimal structure for resources:
     @enduml
```

Comment:
I think the arrows should be pointing the other way.

Reply:
Um, that is supposed to be UML inheritance, see https://en.wikipedia.org/wiki/Unified_Modeling_Language#Diagrams